### PR TITLE
Adds the fireaxe to the black market

### PR DESCRIPTION
**Pull request by MassiveMen**
## Why It's Good For The Game
We need more axe murderer representation in this bitch

## Changelog

:cl:
add: Added the fire axe to the black market uplink

/:cl:

### CODE REVIEW

**Comment by FalloutFalcon** [comment]
Way 2 cheap. Change the description to be something silly like 1 of a kind fireaxe stolen from nt and make it the same price as a high tier gun

**Review by FalloutFalcon — CHANGES_REQUESTED** [review]
What i said above or- make it rusty and nerf the damage

**Comment by generalthrax** [comment]
> Way 2 cheap. Change the description to be something silly like 1 of a kind fireaxe stolen from nt and make it the same price as a high tier gun

Consider: Shiptest's combat scene is dominated by guns. You can buy a shotgun for the same price and deal like 6x times more damage in a quarter of the time. Melee's not quite have the kick it used to.

**Comment by FalloutFalcon** [comment]
> > Way 2 cheap. Change the description to be something silly like 1 of a kind fireaxe stolen from nt and make it the same price as a high tier gun
> 
> 
> 
> Consider: Shiptest's combat scene is dominated by guns. You can buy a shotgun for the same price and deal like 6x times more damage in a quarter of the time. Melee's not quite have the kick it used to.

yes i agree but fireaxes should stay kinda rare and cool. this is way to cheap.
im already working on more melee.

**Comment by MassiveMen** [comment]
> > > Way 2 cheap. Change the description to be something silly like 1 of a kind fireaxe stolen from nt and make it the same price as a high tier gun
> > 
> > 
> > Consider: Shiptest's combat scene is dominated by guns. You can buy a shotgun for the same price and deal like 6x times more damage in a quarter of the time. Melee's not quite have the kick it used to.
> 
> yes i agree but fireaxes should stay kinda rare and cool. this is way to cheap. im already working on more melee.

A price bump of that size would put it at the level of the SUNS saber (actually rare) or the magnetic cleaver (just better). In what universe are fire axes rare.

**Comment by generalthrax** [comment]
Agreed. This is a glorified shipbreaking tool for destroying windows, at best.

**Comment by MassiveMen** [comment]
> Way 2 cheap. Change the description to be something silly like 1 of a kind fireaxe stolen from nt and make it the same price as a high tier gun

As for the joke, sure I could try again, I threw that together at 4am and went to sleep

File: code/modules/cargo/blackmarket/blackmarket_items/weapons.dm [hunk 1]
```diff
@@ -350,4 +350,12 @@
 	stock_max = 2
 	availability_prob = 20
 
+/datum/blackmarket_item/weapon/fireaxe
+	name = "Fireaxe"
+	desc = "A wooden axe handle with a large red blade, it was a tool made to save lives. You're just going to kill people with it though."
+	item = /obj/item/fireaxe
 
+	price_min = 1000
+	price_max = 2000
```
Comment (FalloutFalcon) [inline code/modules/cargo/blackmarket/blackmarket_items/weapons.dm]:
```suggestion
	price_max = 4500
```

**Review by FalloutFalcon — APPROVED** [review]

**Comment by MassiveMen** [comment]
I have no idea what that does I hit it on accident. I'm praying.

File: code/modules/cargo/blackmarket/blackmarket_items/weapons.dm [hunk 2]
```diff
@@ -350,4 +350,12 @@
 	stock_max = 2
 	availability_prob = 20
 
+/datum/blackmarket_item/weapon/fireaxe
+	name = "Fire Axe"
+	desc = "An incredibly sharp axe of dubious make used by various engineers throughout the years to settle arguments while hammered. Sold to us by a very friendly man in a suit."
+	item = /obj/item/fireaxe
```
Comment (FalloutFalcon) [inline code/modules/cargo/blackmarket/blackmarket_items/weapons.dm]:
```suggestion
	item = /obj/item/melee/axe/fire
```

**Review by FalloutFalcon — DISMISSED** [review]

**Pull request merged**